doc/nrf/known_issues.rst

```diff
@@ -327,7 +327,12 @@ nRF5
 nRF5340
 =======
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-0-0
```

Comment:
Probably 2.0.1 and 2.0.2 as derived directly from 2.0.0 are also affected?
```suggestion
.. rst-class:: v2-0-2 v2-0-1 v2-0-0
```

Reply:
Yes they are affected also. Added